helix-view/src/editor.rs

```diff
@@ -52,12 +54,14 @@ impl Default for Config {
 pub struct Editor {
     pub tree: Tree,
     pub documents: SlotMap<DocumentId, Document>,
+    pub path_to_doc: FxHashMap<PathBuf, DocumentId>,
```

Comment:
I'd call this `documents_by_path` and while I think it's useful, you should probably keep the original code which loops through the open documents. This new hashmap doesn't handle possible renames via `set_path` or `set_path + write`